crates/typst/src/layout/transform.rs

```diff
@@ -335,6 +335,115 @@ cast! {
     length: Length => ScaleAmount::Length(length),
 }
 
+/// Skews content without affecting the layout.
+///
+/// Skews an element in horizontal and/or vertical direction. The layout will
+/// act as if the element was not skewed unless you specify `{reflow: true}`.
```

Comment:
Aren't the first and third sentence somehwat contradictory? Since it will affect the layout if `reflow` is set to `true`.

Reply:
Well yes, agreed this is not a clear formulation, I oriented myself at the other transformation functions which have similar descriptions. 

I think I would just remove "without affecting the layout".